chef/cookbooks/hyperv/templates/default/nova.conf.erb

```diff
@@ -7,10 +7,7 @@ cinder_catalog_info=volume:cinder:internalURL
 cinder_api_insecure=<%= @cinder_insecure ? 'True' : 'False' %>
 fake_network=true
 instances_path=<%= @instances_path%>
-<%= "glance_api_servers=#{@glance_server_protocol}://#{@glance_server_host}:#{@glance_server_port}" unless @glance_server_host.nil? %>
-<%= "glance_api_insecure=#{@glance_server_insecure ? 'True' : 'False'}" unless @glance_server_host.nil? %>
 use_cow_images=true
-force_config_drive=false
```

Comment:
There was an error with it, can't recall which, but it's no surprise in openstack